props/prop_base.h

```diff
@@ -1709,7 +1709,7 @@ class PropBase : CommandParser, protected Looper, protected SaberBase, public Mo
     }
     if (!strcmp(cmd, "get_volume")) {
 #ifdef ENABLE_AUDIO
-      STDOUT.println(dynamic_mixer.get_volume());
```

Comment:
Actually. this (and below) should not use PVLOG, because no log level should be able to turn it off. Both should use STDOUT.

Reply:
adjusted.